functions/exercises.py

```diff
@@ -112,10 +126,10 @@ def check_num_in_range(number, range_min, range_max):
     :param range_max:
     :return:
     """
-    # INSERT YOUR CODE HERE
-    #
-    # Run this command to test your implementation:
-    #
+    if number > range_min and number < range_max:
+        return True
+    else:
+        return False

```

Comment:
Maybe this would be a lot simpler
```suggestion
return range_min<number<range_max

```